cypress/e2e/awx/administration/topology-view.cy.ts

```diff
@@ -101,7 +110,7 @@ describe('Topology view', () => {
       });
   });
 
-  it('should show no forks slider when instance has 0 cpu and memory capacity', () => {
+  it.skip('should show no forks slider when instance has 0 cpu and memory capacity', () => {
```

Comment:
I see that you have skipped this test- It looks like it would be a better component test. Perhaps it can be moved to the component test file for this feature, and deleted from this file?